# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wvdial/hotfix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- wvdial-1.60/wvrules.mk.vanilla	2009-02-16 13:19:09.000000000 +0100
+++ wvdial-1.60/wvrules.mk	2009-02-16 13:19:14.000000000 +0100
@@ -60,7 +60,7 @@
 		  -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
 CFLAGS += $(COPTS) $(C_AND_CXX_FLAGS) 
 CXXFLAGS += $(CXXOPTS) $(C_AND_CXX_FLAGS)
-LDFLAGS += $(LDOPTS) -L$(WVSTREAMS_LIB)
+LDFLAGS += $(LDOPTS) # -L$(WVSTREAMS_LIB)
 
 # Default compiler we use for linking
 WVLINK_CC = gcc
--- wvdial-1.60/Makefile.vanilla	2009-02-16 13:14:45.000000000 +0100
+++ wvdial-1.60/Makefile	2009-02-16 13:14:54.000000000 +0100
@@ -30,7 +30,7 @@
 wvdial.a: wvdialer.o wvmodemscan.o wvpapchap.o wvdialbrain.o \
 	wvdialmon.o
 
-wvdial: LDFLAGS+=-luniconf
+wvdial: LDLIBS+=-luniconf
 
 wvdial wvdialconf papchaptest pppmon: wvdial.a
 
